test/intest1.F90

```diff
@@ -34,6 +35,9 @@ program intest1
   call setim8b(.true.)
 #endif
 
+  ! Test the i4dy() function.
+  if (i4dy(80123023) .ne. 1980123023) stop 1
+
```

Comment:
There's already a `stop 1` line further down below in the code, so could you please renumber this to a different stop code, or else leave this and your below `if (idxmsg(1) .ne. 0) stop 2` lines as is and instead just renumber all of the existing stop numbers 1-18 as 3-20?  Or just move these two new tests to the end of the code and number them as 19 and 20?

Either way is fine with me, but the idea is to have a unique number for every stop error, so we always know immediately which test failed, correct?

Reply:
OK